src/App.tsx

```diff
@@ -133,9 +133,27 @@ const TMP_DATA: MarkerFileData = {
 	}]
 };
 
-export default function App() {
+    export default function App() {
+	const [openFileSelector, {filesContent, loading}] = useFilePicker({
+		accept: '.txt',
+	});
+
+	if (loading) {
+		return <div>Loading...</div>;
+	}
+
 	return <div>
 		<h1>Hello, world!</h1>
+		<button onClick={() => openFileSelector()}>Select files</button>
+		<br />
+		{filesContent.map((file, index) => (
+			<div>
```

Comment:
missing key so lint is complaining

not sure we really have one, and I assume this is temporary anyway, so probably just throw in an eslint-disable-next-line